File: webflux/simple-crud/src/main/java/br/com/leonardoferreira/poc/repository/CustomerRepository.java
package br.com.leonardoferreira.poc.repository;

import br.com.leonardoferreira.poc.domain.entity.CustomerEntity;
import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CustomerRepository extends ReactiveMongoRepository<CustomerEntity, String> {
}

File: jpa/jpa-one-to-one/src/main/kotlin/br/com/leonardoferreira/poc/repository/CustomerRepository.kt
package br.com.leonardoferreira.poc.repository

import br.com.leonardoferreira.poc.domain.Customer
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface CustomerRepository : JpaRepository<Customer, Long>
